server/tasks.py

```
@@ -29,19 +32,44 @@
     "tornado": ["#tornado", "#twister"]
 }
 
+
+def _log_task_async(coro):
+    """Helper to run async logging in sync context"""
+    try:
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+        loop.run_until_complete(coro)
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)